merlin-rxjava/src/main/java/com/novoda/merlin/rxjava/MerlinObservable.java

```diff
@@ -0,0 +1,31 @@
+package com.novoda.merlin.rxjava;
+
+import android.content.Context;
+
+import com.novoda.merlin.Merlin;
+import com.novoda.merlin.MerlinBuilder;
+import com.novoda.merlin.NetworkStatus;
+
+import rx.Emitter;
+import rx.Observable;
+
+public class MerlinObservable {
+
+    public static Observable<NetworkStatus.State> from(Context context) {
+        return from(context, new Merlin.Builder());
+    }
+
+    public static Observable<NetworkStatus.State> from(Context context, MerlinBuilder merlinBuilder) {
+        return from(merlinBuilder.withAllCallbacks()
+                                 .build(context));
+    }
+
+    public static Observable<NetworkStatus.State> from(Merlin merlin) {
+        return createObservable(merlin);
+    }
+
+    private static Observable<NetworkStatus.State> createObservable(Merlin merlin) {
+        return Observable.create(new MerlinAction(merlin), Emitter.BackpressureMode.LATEST)
+                         .distinctUntilChanged();
```

Comment:
You're right. It wasn't being done before and I think the library should not be too opinionated on this matter. It's the same case as buffering the events, if the user needs the feature it can be done downstream. Done in c9d27ca